lib/v2/douban/other/discussion.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * @Author: nightmare-mio wanglongwei2009@qq.com
+ * @Date: 2023-11-20 23:36:12
+ * @LastEditTime: 2023-11-21 22:13:01
+ * @Description:
+ */
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const link = 'https://book.douban.com/subject';
+    const { data: response } = await got({
+        method: 'get',
+        url: `${link}/${id}/discussion`,
+    });
```

Comment:
Please use either one style but not both.
https://github.com/DIYgod/RSSHub/blob/d83b6e375844fbca21fff7feebf621b5f7693699/lib/v2/douban/other/discussion.js#L39